extensions/wikia/AdEngine/js/AdSlotInterstitial.js

```diff
@@ -0,0 +1,54 @@
+/*global define*/
+define('ext.wikia.adEngine.slot.interstitial', [
+	'wikia.log',
+	'wikia.window',
+	'wikia.document',
+	'wikia.ui.factory'
+], function (log, win, doc, uiFactory) {
+	'use strict';
+
+	var slotDiv,
+		logGroup = 'ext.wikia.adEngine.slot.interstitial',
+		slotName = 'MODAL_INTERSTITIAL_5',
+		modalConfig = {
+			vars: {
+				id: slotName + '_AD',
+				size: 'medium',
+				content: '<div id="' + logGroup + '.content" class="modal_interstitial ad-centered-wrapper"></div>',
+				title: 'Advertisement',
+				closeText: 'Close',
+				buttons: []
+			}
+		};
+
+	function onAdSuccess() {
+		uiFactory.init('modal').then(function (uiModal) {
+			uiModal.createComponent(modalConfig, function(modal) {
+					doc.getElementById(logGroup + '.content').appendChild(slotDiv);
```

Comment:
object in modal.$content != object that I search for (this is child of modal.$content)